test/versioning/npm.spec.js

```diff
@@ -123,4 +123,88 @@ describe('semver.getNewValue()', () => {
       semver.getNewValue('~0.6.1', 'replace', '0.6.8', '0.7.0-rc.2')
     ).toEqual('~0.7.0-rc');
   });
+  it('handles cap range strategy', () => {
+    [
+      ['=1.0.0', '1.0.0', '1.2.3', '=1.2.3'],
+      ['=1.2', '1.0.0', '1.2.3', '=1.2 <=1.2.3'], // '>=1.2.0 <1.3.0'
+      ['=1', '1.0.0', '1.2.3', '=1 <=1.2.3'], // '>=1.0.0 <2.0.0'
+
+      ['1.0.0 || 1.1.0', '1.0.0', '1.2.3', '1.0.0 || 1.2.3'],
```

Comment:
I think `1.0.0 || 1.1.0 || 1.2.3`. i.e if something is already capped then we should default to our regular behaviour